sklearn/tree/tests/test_tree.py

```diff
@@ -616,7 +616,9 @@ def test_error():
         est.fit([[0, 1, 2]], [5, -0.1, 2])
 
 
-@pytest.mark.parametrize("name, Tree", ALL_TREES.items())
+@pytest.mark.parametrize(
+    "name, Tree", {k: v for k, v in ALL_TREES.items() if "ExtraTree" not in k}
+)
```

Comment:
I think you did the opposite of what you wanted :)
we still want this test for ExtraTrees but not for DecisionTrees
```suggestion
@pytest.mark.parametrize(
    "name, Tree", {k: v for k, v in ALL_TREES.items() if "ExtraTree" in k}
)
```